Fast Algorithm (Corner Detection)/Fast_Algorithm.py

```diff
@@ -0,0 +1,41 @@
+'''
+  For using this Script you need to install OpenCV in your machine
+'''
+#Importing openCV library
+import cv2 as cv
+
+#Taking path of input from the user
+path=input("Enter the path of image: ")
+img=cv.imread(path)
+img=cv.resize(img,(640,640))  #resizing the image
+
+#Printing the original image
+cv.imshow('Original',img)
+
+# Initiate FAST object with default values
+fast = cv.FastFeatureDetector_create()
+
+# find and draw the keypoints
+kp = fast.detect(img,None)
+img2 = cv.drawKeypoints(img, kp, None, color=(255,0,0))
+
+# Print all default parameters
+print( "Threshold: {}".format(fast.getThreshold()) )
+print( "nonmaxSuppression:{}".format(fast.getNonmaxSuppression()) )
+print( "neighborhood: {}".format(fast.getType()) )
+print( "Total Keypoints with nonmaxSuppression: {}".format(len(kp)) )
+
+# Disable nonmaxSuppression
+fast.setNonmaxSuppression(0)
+kp = fast.detect(img,None)
+
+print( "Total Keypoints without nonmaxSuppression: {}".format(len(kp)) )
+final = cv.drawKeypoints(img, kp, None, color=(255,0,0))
+
+#Saving the image
+cv.imwrite('./folder-name/image_name (detected).jpg',final)
```

Comment:
by image name, I mean to say if I passed an image named "abc.jpg", then the output image should be "abc (detected) .jpg" 

Fix this. (Hint: Use f-strings)